packages/ui/src/components/Tabs/tabs.tsx

```diff
@@ -22,6 +22,7 @@ export type TabsProps = Pick<DefaultTabsProps,
 'defaultActiveKey' |
 'activeKey' |
 'destroyInactiveTabPane' |
+'tabBarExtraContent' |
```

Comment:
Use `React.ReactElement` (or ReactNode if something like `[<Item/>, <Item/>, <Item/>]` is allowed), since Antd by default has config (or Node), with `left`, 'right`, `bottom`....props, which is not what we want